classes/Commands/UpdateCommand.php

```diff
@@ -127,17 +129,17 @@ private function chainCommand(OutputInterface $output): int
             return ExitCode::SUCCESS;
         }
 
-        if (strpos($lastInfo, 'bin/console update:start') !== false) {
+        if (strpos($lastInfo, self::$defaultName) !== false) {
```

Comment:
this posed a problem if options were placed before the command (ex: `bin/console -v update:start`)